flareio/api_client.py

```diff
@@ -47,18 +47,21 @@ def _create_session() -> requests.Session:
             ),
         )
 
+        retry = Retry(
+            total=5,
+            backoff_factor=2,
+            status_forcelist=[429, 502, 503, 504],
+            allowed_methods={"GET", "POST"},
+        )
+
+        # Support for urllib3 < 2.X
+        if hasattr(Retry, "backoff_max"):
+            retry.backoff_max = 15
```

Comment:
Can we add an assertion in existing tests that verifies `backoff_max` is set? For example here it could be that you made a typo and we never set it.